certified-connectors/ProcessStreet/apiDefinition.swagger.json

```diff
@@ -518,46 +475,55 @@
         } ],
         "responses" : {
           "200" : {
-            "description" : "The result of the form fields update.",
+            "description" : "The updated workflow run form fields",
             "schema" : {
               "$ref" : "#/definitions/UpdateMultipleFormFieldValuesResponse"
             }
-          },
-          "400" : {
-            "description" : "Invalid value for: body",
+          }
+        }
+      }
+    },
+    "/users" : {
+      "get" : {
+        "summary" : "Get users",
+        "description" : "Retrieves all the users",
+        "operationId" : "ListUsers",
+        "consumes" : [ "application/json" ],
+        "produces" : [ "application/json" ],
+        "parameters" : [ {
+          "name" : "workflowId",
+          "in" : "query",
+          "required" : false,
+          "type" : "string"
+        }, {
+          "name" : "workflowRunId",
+          "in" : "query",
+          "required" : false,
+          "type" : "string"
+        }, {
+          "name" : "formFieldId",
+          "in" : "query",
+          "required" : false,
+          "type" : "string"
+        } ],
```

Comment:
@mwysokinski here too, fix the formatting please so that we can see that the array ends on the new line. The indentation should be correct after these changes.